EESSI-pilot-install-software.sh

```diff
@@ -356,6 +356,12 @@ fail_msg="Installation of WRF failed, that's unexpected..."
 OMPI_MCA_pml=ucx UCX_TLS=tcp $EB WRF-3.9.1.1-foss-2020a-dmpar.eb -r --include-easyblocks-from-pr 2648
 check_exit_code $? "${ok_msg}" "${fail_msg}"
 
+echo ">> Installing R 4.1.0 (better be patient)..."
+ok_msg="R installed, wow!"
+fail_msg="Installation of R failed, so sad..."
+$EB --from-pr 16386 R-4.1.0-foss-2021a.eb --robot --parallel-extensions-install --experimental
```

Comment:
@smoretti Instructing EasyBuild to download the `R` easyconfig from https://github.com/easybuilders/easybuild-easyconfigs/pull/16386 is problematic currently, it fails with:

```
ERROR: Couldn't find path to patched file /tmp/eb-90906wdf/files_pr16386/h/hyperopt/hyperopt-0.2.4-intel-2019b-Python-3.7.4-Java-1.8.eb
```

That's a result of the large number of easyconfigs being touched in this PR, and the changes that were made in https://github.com/easybuilders/easybuild-easyconfigs/pull/16697 (so `hyperopt-0.2.4-intel-2019b-Python-3.7.4-Java-1.8.eb` no longer exists in the `develop` branch).

To some extent, that's a bug in EasyBuild (maybe), but we'll need to work around it.

The most recent change to `R-4.1.0-foss-2021a.eb` that really matters was made in https://github.com/easybuilders/easybuild-easyconfigs/pull/16011 (checksum fix), so let's use that instead?

Reply:
Done

Reply:
I thought using a PR number **and** an eb filename should avoid using all the other eb files from the PR